msteams-platform/whats-new.md

```diff
@@ -482,6 +481,8 @@ Discover Microsoft Teams platform features that are deprecated. You can now get
 
 Teams platform features that aren't available.
 
+* ***April 30, 2024***: Tab margin experience is now deprecated for all types of tabs.
```

Comment:
In tab experiences, tab margins are deprecated.

Reply:
Updated.